projects/igniteui-angular/src/lib/drop-down/drop-down.component.ts

```diff
@@ -505,9 +505,10 @@ export class IgxDropDownComponent extends IgxDropDownBaseDirective implements ID
      * @hidden
      * @internal
      * @param newSelection
+     * @param emit
      * @param event
      */
-    public override selectItem(newSelection?: IgxDropDownItemBaseDirective, event?: Event) {
+    public override selectItem(newSelection?: IgxDropDownItemBaseDirective, emit = true, event?: Event) {
```

Comment:
`emit` could also be last if you don't want to alter any of the existing dropdown code and tests :)
Also there's now an override in the Select for example that's doesn't match the base signature and can in theory be called with the wrong order of params